packages/katello/katello/katello-remove

```diff
@@ -141,38 +164,49 @@ CONTENT.map! do |dir|
 end
 
 def remove
-  puts 'Stopping services'
-  `katello-service stop`
+  puts "\nStopping services".yellow
+  `foreman-maintain service stop`
 
-  puts 'Removing RPMs'
+  puts "\nRemoving RPMs".yellow
```

Comment:
Design wise, it might be a bit cleaner to use semantic naming instead of the colors themselves and avoid opening up the native String class. The example would be calling strings like this:

```
warn("Removing RPMs")
info("")
important("")
```

Not a requirement, but a thought on another way to approach it. I am fine with whichever way you want to go.